### DISCUSSION TITLE
Resolution of stream

### DISCUSSION THREAD

**Discussion by PeteRepeat77** [kjbaker-uk/xctvplayer#14] [General]
I've put my xcloud subscription on pause for a while now but thinking about trying it out again.

As much as there reports keep resurfacing of the official xcloud beta working for some in cgtv, the reports of freezing and the audio glitch are still there so the official app hasn't really developed and each new report is just false hope I feel, so good to see your app may continue development.

Do you have any idea yet what changes you will be targeting for the next revision? Resolution? Controller bug?

**Comment by DVergil** [comment]
I've been suffering from sound issues and image freezes

**Reply by PeteRepeat77** [reply]
With the official android (non tv) app I assume? I found this xctv while needing some improvements doesn't freeze which was a fundamental problem with the official app regardless if how many times it have false hope... Eventually it froze and then sometimes frequent.

**Comment by DVergil** [comment]
A design update would probably improve a few things

**Comment by DVergil** [comment]
Something I have is a problem with the language of the app

**Comment by DVergil** [comment]
I'm Brazilian and here we use xcloud a lot
